src/FormattingToolbar/toolbarTooltip.js

```diff
@@ -28,5 +28,6 @@ export const identifyBlock = (block) => {
   if (typeBeginning === 'bl') return 'Quote';
   if (typeBeginning === 'ul') return 'Bulleted List';
   if (typeBeginning === 'ol') return 'Numbered List';
+  if (block === 'horizontal_rule') return 'Horizontal Rule';
```

Comment:
When `block` contains `'horizontal_rule'` than `typeBeginning` will be `'ho'`.  
I could check against `'ho'` or define a Methode to split the string by an underscore an return the first char of each piece.
That would result in `'hr'`. I used `'horizontal_rule'' before because it's more understandable.